runtime/commonMain/src/kotlinx/serialization/SerialFormat.kt

```diff
@@ -29,6 +29,7 @@ interface BinaryFormat: SerialFormat {
     fun <T> load(deserializer: DeserializationStrategy<T>, bytes: ByteArray): T
 }
 
+// TODO migrate
```

Comment:
migrate to where?

Reply:
to a proper name. Removed this TODO for a while